react/Coupon.tsx

```diff
@@ -78,21 +78,15 @@ const Coupon: StorefrontFunctionComponent = () => {
       {showPromoButton ? (
         <Fragment>
           {!coupon && (
-            <div className="mb5">
-              <Button
-                id="add-coupon"
-                variation="tertiary"
-                collapseLeft
-                noUpperCase
-                onClick={toggle}
-              >
+            <div>
```

Comment:
Do you really need this `<div>`?

Reply:
For some reason, we do. Without the `<div>` the component uses the entire summary width.  See below:

![image](https://user-images.githubusercontent.com/12574426/67984056-bea40600-fc04-11e9-8934-eb4363e88433.png)

Reply:
That happens when the new version of `checkout-summary` (which its gonna be up soon), that uses `flex-layout` on coupon's  block, is linked.